app/src/main/java/app/musikus/ui/library/LibraryScreen.kt

```diff
@@ -543,7 +544,7 @@ fun LibraryUiItem(
                     },
                 )
                 Text(
-                    text = "last practiced: yesterday",
+                    text = "last practiced: " +  (lastPracticedDate?.musikusFormat(DateFormat.DAY_MONTH_YEAR) ?: "never"),
```

Comment:
Is this really the right place for human-readable conversion of `ZonedDateTime` object? What if we want to add more logic, e.g. "yesterday", "last sunday", "today" in the future? Wouln't be the ViewModel the right place to implement this *UI logic*?

Reply:
This will be solved with #4